platform/platform.hpp

```diff
@@ -121,6 +121,7 @@ class Platform
 
   /// @return true if we can create custom texture allocator in drape
   static bool IsCustomTextureAllocatorSupported();
+  static string GetCurrentWorkingDirectory() noexcept;
```

Comment:
Please, add a comment to the function, i.e. when it returns empty string.

Reply:
Done